ui/src/app/components/help-sidebar.tsx

```diff
@@ -237,6 +248,25 @@ const icons = (helpContentKey: string) => {
       style: {fontSize: '20px', marginLeft: '3px'},
       tooltip: 'Annotations',
     }];
+  if (
+    enableCustomRuntimes
+    && (
+      workspaceAccessLevel === WorkspaceAccessLevel.OWNER
+      || workspaceAccessLevel === WorkspaceAccessLevel.WRITER
+    )
+  ) {
+    return [...iconsList, {
+      id: 'thunderstorm',
+      disabled: true,
+      faIcon: null,
+      label: 'Cloud Icon',
+      page: null,
+      style: {height: '22px', width: '22px', marginTop: '0.25rem', opacity: 0.5},
```

Comment:
It looks sort of grayed out - is the opacity the issue?

Reply:
Ah, yes, it is. I initially assumed it had to do with the disabled attribute. I can make it actually condition on that or I can just not since it's behind a feature flag and we're about to implement the panel content anyways

Reply:
Yeah - I think just drop it and make this undisabled, looks like some other styling gets added if the icon is disabled, though I don't totally follow the code. 

I also think there's an argument to make the disabled status contingent on writer access, rather than hiding the icon. This isn't clearly spec'd out in the PRD or mocks, but I think most of our existing UI patterns disable the control if you lack permissions.

Reply:
The styling is just changing the cursor to a circle with a slash through it.

I agree that there's an argument for conditioning the disabled status on writer access rather than hiding the icon, but I did run this by both Shimon and Lou and they both prefer to have the icon hidden if the user isn't at least a writer.

Reply:
OK, cool, didn't realize you had already covered this with Lou and Shimon. Sounds good - then I'd just hardcode it to `disabled: false` for now and drop the opacity styling.